src/main/java/frc/robot/RobotContainer.java

```diff
@@ -57,6 +67,13 @@ public RobotContainer() {
    */
   private void configureButtonBindings() {
     driveController = new XboxController(driveControllerPort);
+    altController = new XboxController(altControllerPort);
+
+    JoystickButton toggleIntake = new JoystickButton(altController, Button.kX.value);
+    toggleIntake.toggleWhenPressed(new RunIntake(IntakeConstants.intakeMotorPower));
```

Comment:
change to defaultMotorPower.
Also import static Constants.IntakeConstants so you dont have to do IntakeConstants. every time